map.js

```diff
@@ -436,8 +379,10 @@ async function initImgs(){
 }
 
 function onMouseClick(mouseLoc){
-    console.log("click");
-    let handled = topbar.click(mouseLoc);
+    if(window.debug){
```

Comment:
I'm putting more of my statements behind window.debug checks like this, so that in your main file, you can just set window.debug =true, and then all the logging turns on, and when it is false, users don't get spooked from all the diagnostic warnings.

Reply:
sounds like a good system. will start using.